azurerm/internal/services/portal/portal_tenant_configuration_resource.go

```diff
@@ -0,0 +1,121 @@
+package portal
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/Azure/azure-sdk-for-go/services/preview/portal/mgmt/2019-01-01-preview/portal"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/helpers/tf"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/clients"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/services/portal/parse"
+	azSchema "github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/tf/schema"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/timeouts"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/utils"
+)
+
+func resourceTenantConfiguration() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceTenantConfigurationCreateUpdate,
+		Read:   resourceTenantConfigurationRead,
+		Update: resourceTenantConfigurationCreateUpdate,
+		Delete: resourceTenantConfigurationDelete,
+
+		Timeouts: &schema.ResourceTimeout{
+			Create: schema.DefaultTimeout(30 * time.Minute),
+			Read:   schema.DefaultTimeout(5 * time.Minute),
+			Update: schema.DefaultTimeout(30 * time.Minute),
+			Delete: schema.DefaultTimeout(30 * time.Minute),
+		},
+
+		Importer: azSchema.ValidateResourceIDPriorToImport(func(id string) error {
+			_, err := parse.TenantConfigurationID(id)
+			return err
+		}),
+
+		Schema: map[string]*schema.Schema{
+			"enforce_private_markdown_storage": {
```

Comment:
to keep consistent with other bools in the provider
```suggestion
			"private_markdown_storage_enforced": {
```

Reply:
Updated